docs/source/algorithms/PolarizationCorrectionWildes-v1.rst

```diff
@@ -153,6 +153,8 @@ References
 .. [#WILDES] A. R. Wildes, *Neutron News*, **17** 17 (2006)
              `doi: 10.1080/10448630600668738 <https://doi.org/10.1080/10448630600668738>`_
 
+.. seealso :: Algorithm :ref:`algm-PolarizationEfficienciesWildes`
+
```

Comment:
I'd be lying if I said I really understand how this is working 🙂 I noticed something similar seems to be in use in other documentation i.e. https://github.com/mantidproject/mantid/blob/main/docs/source/algorithms/ReflectometryISISLoadAndProcess-v1.rst?plain=1#L122, but that one is a Python algorithm. I'll take another look and see if I can figure out how this needs to be set up.

Reply:
It's collected by the `.. relatedalgorithms::` directive at the top of the file. But including a `seealso::` overrides that and stops it from appearing properly. [The code for the directive](https://github.com/mantidproject/mantid/blob/main/docs/sphinxext/mantiddoc/directives/relatedalgorithms.py) basically just calls the agorithm's `seeAlso()` method and then sorts out the links.

Reply:
Great, thanks, and do you know why it seems to be a bit different for Python algorithms? I noticed a few docs files for Python algorithms that seem to include the seealso line in the rst file?

Reply:
I think it's just a copying thing. For instance [HFIRSANS2Wavelength](https://docs.mantidproject.org/nightly/algorithms/HFIRSANS2Wavelength-v1.html) is a python algorithm that uses the `SeeAlso()` method and directive and works just like the C++ ones.

Reply:
I found that the issue with the documentation in this PR was that neither had the `.. relatedalgorithms::` directive at the top, which as you say is what is actually required to pull through the `seeAlso()` algorithms. It didn't seem to make any difference whether I also included the `.. seealso` directive, it worked with or without (which explains why some `.rst` files that contain both still work, like the one we have for `ReflectometrISISLoadAndProcess`). I'll remove the `.. seealso` directive from the docs in this PR, though, because it's obsolete even if it may not be causing any issues.

Reply:
Thanks, I'd missed that I'd also added the line in the Wildes workspace order PR, that's good to know.